atlp-rwanda/eagles-ec-fe
# #187419115 Configure ESLint and Git Pre commit hook to run tests and ESlint format

#### What does this PR do?

This Configures Eslint and git precommit hook to run tests and lint before commits and adds Hound CI

#### Description of Task to be completed?

- configure eslint
- configure prettier for code formatting
- Configure Hound CI and add a badge in Readme

#### How should this be manually tested?
1. Clone the repository.
2. Checkout to the branch `ch-eslint-precommit-#187419115`.
3. Run `npm install` to install dependencies.
4. Add any changes and try committing to check for precommit hook
5. Run `npm run lint` to check lint errors and `npm run lint:fix` to fix lint errors
6. Run `npm run prettier` to format codes with prettier ( you don't need to have prettier installed )

#### What are the relevant pivotal tracker stories?

Delivers #187419115
